front/pages/api/w/[wId]/index.ts

```diff
@@ -103,6 +114,20 @@ async function handler(
         });
 
         owner.ssoEnforced = body.ssoEnforced;
+      } else if ("whiteListedProviders" in body) {
+        if (body.whiteListedProviders.every(isModelProviderId)) {
```

Comment:
We should be able to remove this check and the one on isModelProviderId if we use the valid types on `WorkspaceProvidersUpdateBodySchema` & `WorkspaceEmbedderUpdateBodySchema`?